Source/WebKit/UIProcess/Automation/WebAutomationSession.h

```diff
@@ -117,7 +118,8 @@ class WebAutomationSession final : public API::ObjectImpl<API::Object::Type::Aut
     void setSessionIdentifier(const String& sessionIdentifier) { m_sessionIdentifier = sessionIdentifier; }
     String sessionIdentifier() const { return m_sessionIdentifier; }
 
-    WebProcessPool* processPool() const { return m_processPool; }
+    WebProcessPool* processPool() const { return const_cast<WebProcessPool*>(m_processPool.get()); }
```

Comment:
Why do we want to keep this one, shouldn't everyone use the one that returns a RefPtr?

Reply:
Oh these are useful when we have chaining like: foo()->bar()->baz() but nobody does that here so we can get rid of this version indeed.